app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -13,6 +10,44 @@ module Pacbio
     # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
     # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package
     # for the service implementation of the JSON:API standard.
+    # This resource represents a Pacbio Pool and can return all pools, a single pool or multiple
+    # pools along with their relationships.
+    #
+    # ## Filters:
+    #
+    # * sample_name
+    # * barcode
+    #
+    # ## Primary relationships:
+    #
+    # * tube
+    # * used_aliquots
+    # * primary_aliquot
+    # * requests
+    # * libraries
```

Comment:
You could link these to the resource (or model) by adding {V1::Pacbio::LibraryResource} which makes traversing the docs slightly easier.